roles/install_dbserver/tasks/EPAS_RedHat_install.yml

```diff
@@ -70,3 +70,15 @@
     state: present
   become: true
   when: pg_ssl
+
```

Comment:
Yeah, accidentally left out, but amended and push commit to include. `roles/manage_operating_system/tasks/enable_core_dump.yml`.

Reply:
Also, with the new `roles/manage_operating_system/tasks/enable_core_dump.yml`, do you plan on importing these tasks into a benchmark playbook? I think there should still be a way into `enable_core_dump.yml` through the `roles/manage_operating_system/tasks/main.yml` 
```
- name: Enable core dumps
  ansible.builtin.include_tasks: enable_core_dump.yml
  when:
    - enable_core_dump | bool
```
and add the `enable_core_dump: false` var into the `roles/manage_operating_system/defaults/main.yml` to keep consistent with `enable_user_profiling`

Reply:
> if the `yum-utils` is installed in the `manage_operating_system` tasks, should the debuginfo packages also be installed in the same place? I don't see the need to break it up

I think this might be a chicken and egg issue.  We can't know what debug packages to install until we know what packages to install, but we need to have the tool to install debug packages before we try.

At least I think it makes sense to install the tool so that we can install debug packages, and the install them from the respective playbooks when it's known what the packages are.

I'm open to other ideas.

Reply:
> - name: Enable core dumps
>   ansible.builtin.include_tasks: enable_core_dump.yml
>   when:
>     - enable_core_dump | bool

Whoops, thanks for catching that.  That's what I intended but I think I lost of changes swapping between system.  Amended commit and forced push.

Reply:
what if we kept it here but also placed it in the `install_dbserver` role? Like this in `install_dbserver` and keep the current `yum-utils` install in `manage_operating_system` because if it is already installed, this is not expensive to run and will be skipped? 
```
- name: Install debuginfo helper packages when enable_core_dumps
  ansible.builtin.package:
    name: yum-utils
    state: present
  when:
    - enable_core_dumps is defined
    - enable_core_dumps | bool
  become: true
```

Reply:
> what if we kept it here but also placed it in the `install_dbserver` role? Like this in `install_dbserver` and keep the current `yum-utils` install in `manage_operating_system` because if it is already installed, this is not expensive to run and will be skipped?

Ok, I can put it back.  I don't see the harm, I'm not sure if it's useful without the other operating system changes, but we can see how it goes.

Reply:
i understand, but I think including so that nothing fails here if this is run prior to the `manage_operating_system` role is useful